x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots.ts

```diff
@@ -83,14 +117,20 @@ export function screenshotsObservableFactory(server) {
     );
   };
 
-  const getNumberOfItems = async (browser, layout) => {
+  const getNumberOfItems = async (
+    browser: HeadlessBrowser,
+    layout: LayoutInstance
+  ): Promise<number> => {
     // returns the value of the `itemsCountAttribute` if it's there, otherwise
     // we just count the number of `itemSelector`
     const itemsCount = await browser.evaluate({
       fn: (selector, countAttribute) => {
         const elementWithCount = document.querySelector(`[${countAttribute}]`);
-        if (elementWithCount) {
-          return parseInt(elementWithCount.getAttribute(countAttribute));
+        if (elementWithCount && elementWithCount != null) {
+          const count = elementWithCount.getAttribute(countAttribute);
+          if (count && count != null) {
```

Comment:
same here, isn't the null check redundant?

Reply:
trickier here because `count` could be "0" which is falsey

Reply:
https://github.com/elastic/kibana/pull/42102/commits/020f2a6c8fdfa4209af2b459152956553cd7d450